### PR TITLE
024 Script for requirement [PerformAudioPassThru] Speak (SUCCESS) + UI.PerformAudioPassThru (<applicable_resultCode>)

### PR DESCRIPTION
**Pull request by maria-mih**
024_ATF_P_PerformAudioPassThru_TTS_SUCCESS_UI_success_true.lua

All common functions used available at #703
Testing coverage available at #643 as well as https://github.com/smartdevicelink/sdl_atf_test_scripts/blob/feature/extended_policy_functionality/test_sets/PAPT_new_param_audioPassThruIcon.txt

@istoimenova Please review

### CODE REVIEW

File: test_scripts/API/PerformAudioPassThru/024_ATF_P_PerformAudioPassThru_TTS_SUCCESS_UI_success_true.lua [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+	      maxDuration = 2000,
+	      bitsPerSample = "8_BIT",
+	      audioType = "PCM",
+	      muteAudio = true,
+	      audioPassThruIcon =
+		    { 
+		      value = "icon.png",
+		      imageType = "STATIC"
+		    }
+	    })
```
Comment (istoimenova) [inline test_scripts/API/PerformAudioPassThru/024_ATF_P_PerformAudioPassThru_TTS_SUCCESS_UI_success_true.lua]: @maria-mih, please add one empty row to be more readable.

File: test_scripts/API/PerformAudioPassThru/024_ATF_P_PerformAudioPassThru_TTS_SUCCESS_UI_success_true.lua [hunk 2]
```diff
@@ -0,0 +1,168 @@
...
+	        value = "icon.png"
+	      }
+	    })
+	  :Do(function(_,data)
+
+	  	local function UIPerformAudioResponse()
+	  	self.hmiConnection:SendResponse(data.id, data.method, hmi_result_code[i].result_code) 
+	    end
+	    RUN_AFTER(UIPerformAudioResponse, 1500)
+  end)
```
Comment (istoimenova) [inline test_scripts/API/PerformAudioPassThru/024_ATF_P_PerformAudioPassThru_TTS_SUCCESS_UI_success_true.lua]: @maria-mih, please allign rows 136-152.

File: test_scripts/API/PerformAudioPassThru/024_ATF_P_PerformAudioPassThru_TTS_SUCCESS_UI_success_true.lua (hunk 1 above)
Comment (maria-mih) [inline test_scripts/API/PerformAudioPassThru/024_ATF_P_PerformAudioPassThru_TTS_SUCCESS_UI_success_true.lua]: @istoimenova Added

File: test_scripts/API/PerformAudioPassThru/024_ATF_P_PerformAudioPassThru_TTS_SUCCESS_UI_success_true.lua (hunk 2 above)
Comment (maria-mih) [inline test_scripts/API/PerformAudioPassThru/024_ATF_P_PerformAudioPassThru_TTS_SUCCESS_UI_success_true.lua]: @istoimenova Aligned

**Comment by maria-mih** [comment]
@istoimenova Changes made on commit c4fb41a542d4b031f73110fb1c5dbf247cbd1120

**Pull request merged**